project/Dependencies.scala

```diff
@@ -256,12 +258,11 @@ object Dependencies {
     TestDependencies.commonsIo, // to force TestDependencies.dockerClient to use safe version of this lib
     TestDependencies.commonsMath,
     TestDependencies.httpClient, // to force TestDependencies.dockerClient to use safe version of this lib
```

Comment:
we can follow up in a 2nd PR but a lot of these dependencies are there to force dockerClient to use newer dependencies - they can probably be removed - yet we may that docker-java has 1 or 2 old dependencies of its own (but not nearly as bad as dockerClient)

Reply:
Updated by removing the deps forced by docker-client